tests/test_utils.py

```diff
@@ -261,6 +260,7 @@ def callback_retries(expected_num_retries):
 
     def test_get_superflore_version(self):
         """Test get SuperFlore version"""
-        if __version__ != 'unset':
-            self.assertGreaterEqual(parse_version(get_superflore_version()),
-                                    parse_version('0.2.1'))
+        if VERSION != 'unset':
+            self.assertRegex(get_superflore_version(),
+                             r"^(\d{2}|[1-9]|0\.(\d{2}|[4-9]|3\.(\d{2}|[4-9]|3)))",
```

Comment:
Surprised (and a little disappointed) that my use of a cursed regexp to do the version parsing went unremarked.